swagger.yaml

```diff
@@ -447,6 +447,35 @@ paths:
           description: "Invalid request body or unexpected Cognito response"
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /groups-report:
+    get:
+      tags:
+        - Groups
+      summary: "List groups and the membership of the groups"
+      description: "
+      If a preview team ( cognito group ) is empty of users then this is not on the report.  
+      If the preview team ( cognito group ) acts as a role (i.e. publishers, admin) then this is included on the report
+      if header accept "
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - in: header
+          name: Accept
+          type: string
+          required: true
+          default: application/json
+          description: if text/csv will return csv list else json
+      responses:
+        200:
+          description: "The list of groups and members"
+          schema:
+            $ref: '#/definitions/GroupsUsersList'
+        500:
+          description: "Internal error occurred"
+          schema:
+            $ref: '#/definitions/ErrorResponse'
```

Comment:
In the code there's possibilities of a 404 at the moment (which I'm not sure about - see groups.go)